src/behavior.js

```diff
@@ -262,6 +256,9 @@ export default class Behavior extends React.PureComponent {
     }
 
     const addProp = (prop, defaultValue) => {
+      if (freeze) {
+        return (state[this.key] && state[this.key][prop]) ?? defaultValue
+      }
```

Comment:
```suggestion
      if (freeze) {
        return (state[this.key] && state[this.key][prop]) || defaultValue
      }
      
```

- no support for new JS in this repo so use `||`
- add an empty line after the block for styling

Reply:
done here [c860095](https://github.com/ExodusMovement/haraka/pull/16/commits/c860095e67fb75a8b3bc190c4477e9fc4b14339d)

Reply:
I made some readjustment to better reflect the behavior of `??` on this commit [0ce46f8](https://github.com/ExodusMovement/haraka/pull/16/commits/0ce46f85d31b3106507835956e15fd5fe7640f86). Using `||` operator would result in wrong output when `state[this.key][prop] = 0`.